canonical-web-and-design/cn.ubuntu.com
# add dynamic internet of things page and data

### Done

Made the /internet-of-things page use the CMS
### QA

`make import-new-pages`
`make run`
Navigate to /internet-of-things and see that the correct content is there
### Card

https://canonical.leankit.com/Boards/View/111185042/118530726
